content/blog/2019-02-20-react-translations.md

```diff
@@ -0,0 +1,75 @@
+---
+title: "¡React en español! And Other Languages"
+author: [tesseralis]
+---
+
+We’re excited to announce an ongoing effort to maintain official translations of the React documentation website into different languages. Thanks to the dedicated efforts of React community members from around the world, React is now being translated into *over 30* languages! You can check out our ongoing efforts in the new [Languages](/languages) page!
```

Comment:
/languages is a dead link?

also I hate to be the exclamation point police but this is three in a row… maybe drop this middle one?